src/renderers/dom/client/wrappers/DisabledInputUtils.js

```diff
@@ -30,16 +33,23 @@ var disableableMouseListenerNames = {
  * when `disabled` is set.
  */
 var DisabledInputUtils = {
-  getNativeProps: function(inst, props) {
-    if (!props.disabled) {
-      return props;
+  assignWithNativeProps: function(a,b,c,d,e,f,g) {
+    invariant(!g, 'assignWithNativeProps only accepts 6 arguments');
```

Comment:
This doesn't feel awesome, but I wanted to lean on `assign` as before and didn't want to `slice` arguments into another allocation.